diff --git a/example/TrueSDKSample.xcodeproj/project.pbxproj b/example/TrueSDKSample.xcodeproj/project.pbxproj
index d4dbeae..31f2cc2 100644
--- a/example/TrueSDKSample.xcodeproj/project.pbxproj
+++ b/example/TrueSDKSample.xcodeproj/project.pbxproj
@@ -325,7 +325,7 @@
 				);
 				INFOPLIST_FILE = TrueSDKSample/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = "YOUR-BUNDLE-ID";
+				PRODUCT_BUNDLE_IDENTIFIER = com.truecaller.SwiftTrueSDKHost;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_VERSION = 3.0;
 				TARGETED_DEVICE_FAMILY = 1;
@@ -344,7 +344,7 @@
 				);
 				INFOPLIST_FILE = TrueSDKSample/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = "YOUR-BUNDLE-ID";
+				PRODUCT_BUNDLE_IDENTIFIER = com.truecaller.SwiftTrueSDKHost;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_VERSION = 3.0;
 				TARGETED_DEVICE_FAMILY = 1;
diff --git a/example/TrueSDKSample/AppDelegate.swift b/example/TrueSDKSample/AppDelegate.swift
index 6cb6c76..09d87ce 100644
--- a/example/TrueSDKSample/AppDelegate.swift
+++ b/example/TrueSDKSample/AppDelegate.swift
@@ -25,7 +25,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
              SET THE APP_LINK VALUE IN THE ENTITLEMENTS FILE AS WELL!!!
              UPDATE THE APPID (BUNDLE IDENTIFIER) IN YOUR PROJECT SETTINGS!!!
             */
-            TCTrueSDK.sharedManager().setup(withAppKey: <#API_KEY#>, appLink: <#APP_LINK#>)
+            TCTrueSDK.sharedManager().setup(withAppKey: "KWuTo3267b73e2bcb43c3a0ff4ed6c0be85aa", appLink: "https://a388b96541ab46159681eefde4094ff3.truecallerdevs.com")
         }
         
         return true
diff --git a/example/TrueSDKSample/TrueSDKSample.entitlements b/example/TrueSDKSample/TrueSDKSample.entitlements
index a4cf3ef..ab2577b 100644
--- a/example/TrueSDKSample/TrueSDKSample.entitlements
+++ b/example/TrueSDKSample/TrueSDKSample.entitlements
@@ -4,7 +4,7 @@
 <dict>
 	<key>com.apple.developer.associated-domains</key>
 	<array>
-		<string>applinks:APP_LINK</string>
+		<string>applinks:a388b96541ab46159681eefde4094ff3.truecallerdevs.com</string>
 	</array>
 </dict>
 </plist>
